susemanager/src/mgr_bootstrap_data.py

```diff
@@ -117,7 +117,6 @@
     "python-pyasn1",
     "python-pycparser",
     "python-pyOpenSSL",
-    "python-setuptools",
```

Comment:
Removing from `PKGLIST12`...

Reply:
OK because:

    'SLE-12-*-*' : {
        'PDID' : 1421, 'PKGLIST' : PKGLIST12 + ONLYSLE12 + ENHANCE12SP1,
                                                                                          ^^^^^^^^^^